app/jobs/report_attachment_job.rb

```diff
@@ -0,0 +1,6 @@
+class ReportAttachmentJob < ApplicationJob
+  def perform(ticket_id, work)
+    download = Download.new(work, mime_type: "text/html").generate
+    reporter.send_attachment!(ticket_id, download)
```

Comment:
I'm not sure this will actually work; `generate` returns the `Download` object itself, not an HTML string or a file. I think you'd need to do something like this to get the generated file:
```suggestion
    download = Download.new(work, mime_type: "text/html")
    html = DownloadWriter.new(download).generate_html
    reporter.send_attachment!(ticket_id, html)
```
or call `.generate` and do `File.open` on the path it creates